pack/commands.cfg

```diff
@@ -51,3 +51,7 @@ define command{
        command_name     check_host_services_running
        command_line     $PLUGINSDIR$/check-rhv/check_rhv_main.py -R $HOSTALIAS$ -u $_HOSTRHVM_LOGIN$ -p $_HOSTRHVM_PASSWORD$ -m services_status -s "{'vdsmd': 'Active: active (running)', 'ovirt-ha-agent': 'Active: active (running)'}"
 }
+define command{
+       command_name     check_template_count
```

Comment:
Also needs an associated service that uses this command

Reply:
Added